.gitignore

```diff
@@ -31,3 +31,4 @@ yarn-debug.log*
 .yarn-integrity
 .idea/
 .idea/workspace.xml
+
```

Comment:
@Mubasher112  this change is not required, please remove it. you added an empty line :)